regression/regression_utils.py

```diff
@@ -31,6 +32,38 @@
 """
 
 
+class RetryTestsMetaclass(type):
+
+    NUM_RETRIES = 2
+    FLAKY_ERROR_CLASSES = (AssertionError,)
```

Comment:
I guess I'd like to see a case where we had an assertion failure (rather than a network outage, timeout, etc.) which had a reasonable chance of succeeding if retried immediately:  if the issue is due to "eventual" consistency, I don't know how long we should be waiting.  I'm not even sure that regression testing the "eventual" stuff is worth the trouble.